nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -611,6 +613,117 @@ def _plot_surf_contours(
+
+        cbar_h = 0.25
+        title_h = 0.25 * (title is not None)
+        w, h = plt.figaspect((len(modes) + cbar_h + title_h) / len(hemis))
+        fig = plt.figure(figsize=(w, h), constrained_layout=False)
+        height_ratios = [title_h] + [1.0] * len(modes) + [cbar_h]
+        grid = GridSpec(
+            len(modes) + 2,
+            len(hemis),
+            left=0.0,
+            right=1.0,
+            bottom=0.0,
+            top=1.0,
+            height_ratios=height_ratios,
+            hspace=0.0,
+            wspace=0.0,
+        )
+        axes = []
+
+        for i, (mode, hemi) in enumerate(itertools.product(modes, hemis)):
+            bg_map = None
+            # By default, add curv sign background map if mesh is inflated,
+            # sulc depth background map otherwise
+            if inflate:
+                curv_map = self.load_surf_data(surf_mesh[f"curv_{hemi}"])
+                curv_sign_map = (np.sign(curv_map) + 1) / 4 + 0.25
+                bg_map = curv_sign_map
+            else:
+                sulc_map = surf_mesh[f"sulc_{hemi}"]
+                bg_map = sulc_map
+
+            ax = fig.add_subplot(grid[i + len(hemis)], projection="3d")
+            axes.append(ax)
+
+            self.plot_surf_stat_map(
+                surf[hemi],
+                texture[hemi],
+                view=mode,
+                hemi=hemi,
+                bg_map=bg_map,
+                bg_on_data=bg_on_data,
+                axes=ax,
+                colorbar=False,  # Colorbar created externally.
+                vmin=vmin,
+                vmax=vmax,
+                threshold=threshold,
+                cmap=cmap,
+                symmetric_cbar=symmetric_cbar,
+                **kwargs,
+            )
+
+            # We increase this value to better position the camera of the
+            # 3D projection plot. The default value makes meshes look too
+            # small.
+            ax.set_box_aspect(None, zoom=1.3)
+
+        if colorbar:
+            sm = _colorbar_from_array(
+                self.get_image_data(stat_map),
+                vmin,
+                vmax,
+                threshold,
+                symmetric_cbar=symmetric_cbar,
+                cmap=plt.get_cmap(cmap),
+            )
+
+            cbar_grid = GridSpecFromSubplotSpec(3, 3, grid[-1, :])
+            cbar_ax = fig.add_subplot(cbar_grid[1])
+            axes.append(cbar_ax)
+            # Get custom ticks to set in colorbar
+            ticks = _get_ticks(vmin, vmax, cbar_tick_format, threshold)
+            fig.colorbar(
+                sm,
+                cax=cbar_ax,
+                orientation="horizontal",
+                ticks=ticks,
+                format=cbar_tick_format,
+            )
+
+        if title is not None:
+            fig.suptitle(
+                title, y=1.0 - title_h / sum(height_ratios), va="bottom"
+            )
+
+        return save_figure_if_needed(fig, output_file, bbox_inches="tight")
```

Comment:
I've updated `save_figure_if_needed` with `bbox_inches` parameter to be able to use it here as well.